src/OAuth2User.ts

```diff
@@ -162,6 +161,10 @@ export class OAuth2User implements OAuthClient {
       code_challenge_method = "plain";
     }
     const code_challenge = this.code_challenge;
+    if(options.testnet){
+      AUTHORIZE_URL="https://app.regtest.getalby.com/oauth"
+
+    }
     const url = new URL(AUTHORIZE_URL);
```

Comment:
I think `options.authorizeUrl || AUTHORIZE_URL` is better, so we can keep the existing one as a constant and default value

Reply:
resolved